openmc/data/thermal.py

```diff
@@ -676,6 +676,15 @@ def from_ace(cls, ace_or_filename, name=None):
                     # here, because they are equiprobable, so the order
                     # doesn't matter.
                     mu.sort()
+
+                    # Older versions of NJOY had a bug, and the discreet
+                    # scattering angles could sometimes be less than -1 or
+                    # greater than 1. We check for this here, and warn users.
+                    if mu[0] < -1. or mu[-1] > 1.:
```

Comment:
IMO we should just go ahead and check if `np.any(np.abs(mu) > 1.0)` and warn if so, out of an abundance of caution.

Reply:
This was certainly a problem before, as I was finding many files with discrete angles which were not necessarily in ascending order. That was likely causing bad approximations to be made with the angle-smearing operation to "smooth" out the distribution, so I added a sort on mu in the previous PR. Now that mu is guaranteed to be in ascending order, I think just checking the first and last elements should suffice ?

Reply:
OK, makes sense!